projects/lib/src/board/westernboard.cpp

```diff
@@ -496,6 +510,15 @@ QString WesternBoard::castlingRightsString(FenNotation notation) const
 	return str;
 }
 
+bool WesternBoard::pawnAmbiguity( StepType t ) const
+{
+	int count = 0;
+	for (auto a: pawnSteps)
```

Comment:
Yeah, the implicitly shared Qt containers haven't aged so well: https://www.kdab.com/goodbye-q_foreach/